src/network/api/contact-action.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { ErrorSoapBodyResponse, soapFetch } from '@zextras/carbonio-shell-ui';
+
+import { GenericSoapPayload } from './types';
+import { NAMESPACES } from '../../constants/api';
+
+export const CONTACT_ACTION_OPERATION = {
+	delete: 'delete',
```

Comment:
This enumeration is not correct for the ContactAction. I used the operations for the distribution lists, sorry. I will push a commit with the correct operations.